fdbclient/admin_client.go

```diff
@@ -318,7 +309,30 @@ func (client *cliAdminClient) getStatus(useClientLibrary bool) (*fdbv1beta2.Foun
 	if err != nil {
 		return nil, err
 	}
-	client.log.V(1).Info("Fetched status JSON", "status", status)
+
+	client.log.V(1).Info("Fetched status JSON with fdbcli", "version", client.Cluster.GetRunningVersion(), "status", status)
```

Comment:
```suggestion
	client.log.V(1).Info("Fetched status JSON with fdbcli", "version", client.Cluster.GetRunningVersion())
```

We already log the full status at the return of the GetStatus method.